### PR TITLE
Fixing broken styles

### PR DESCRIPTION
**Pull request by kaytcampbell**
some previous changes broke some styles on membership, this fixes those.

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 1]
```diff
@@ -117,7 +111,7 @@ span.nav-icon{
   margin: $base-margin*2;
   margin-top: $base-margin*4;
   font-size: $base-font-size*1.5;
-  color: $smoke;
+  color: rgb(179, 179, 179);
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: Is $smoke too light for this? If we want to make a new grey shade we should probably make a new colour variable for it.

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 2]
```diff
@@ -6,11 +6,9 @@
   background: $white;
   padding: $base-padding*3;
 }
-
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: Should have a space here.

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 3]
```diff
@@ -6,11 +6,9 @@
...
 .dashboard-title:first-child{
   @extend .mt0;
 }
-
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: Should have a space here.

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 4]
```diff
@@ -59,7 +57,6 @@
       opacity: 1;
       margin-top: 0;
     }
-
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: Should have a space here.

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 5]
```diff
@@ -85,7 +82,6 @@
   margin-left: 74px;
   margin-top: -$base-margin*3;
   list-style: none;
-
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: Should have a space here.

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 6]
```diff
@@ -95,12 +91,10 @@
     font-size: $base-font-size*0.875;
   }
 }
-
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: Should have a space here.

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 7]
```diff
@@ -95,12 +91,10 @@
...
 a.activepage{
   background: $brand-blue-light;
   color: $white;
 }
-
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: Should have a space here.

**Comment by kathleenAMA** [comment]
:hankey: Just a few syntax things

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss (hunk 1 above)
Comment (kaytcampbell) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: I think smoke looks fine, its not that much lighter than the rgb value 
image

**Comment by kaytcampbell** [comment]
:wave:

**Comment by kathleenAMA** [comment]
:art: :art: :art: :elephant: :elephant: :elephant:

**Pull request merged**